util/errors/errors.go

```diff
@@ -112,6 +112,9 @@ func isTransientNetworkErr(err error) bool {
 	} else if strings.Contains(errorString, "connect: connection refused") {
 		// If err is connection refused, retry.
 		return true
+	} else if strings.Contains(errorString, "invalid connection") {
```

Comment:
Is this a common error or is it a vendor specific error?

Reply:
looks like a common error https://stackoverflow.com/questions/65253798/go-sql-driver-get-invalid-connection-when-wait-timeout-is-8h-as-default